src/plugins/dashboard/public/dashboard_container/component/grid/dashboard_grid.test.tsx

```diff
@@ -102,15 +93,16 @@ test.skip('DashboardGrid removes panel when removed from container', () => {
     </DashboardServicesProvider>
   );
 
-  const originalPanels = props.container.getInput().panels;
-  const filteredPanels = { ...originalPanels };
-  delete filteredPanels['1'];
-  props.container.updateInput({ panels: filteredPanels });
+  // const originalPanels = props.container.getInput().panels;
+  // const filteredPanels = { ...originalPanels };
+  // delete filteredPanels['1'];
+  // props.container.updateInput({ panels: filteredPanels });
   component.update();
   const panelElements = component.find('EmbeddableChildPanel');
   expect(panelElements.length).toBe(1);
 });
 
+// TODO PORTABLE DASHBOARSD: Unskip these tests
```

Comment:
I've updated these tests, so that they should work with the new system, and so there are no more lines commented out. That said, these have been skipped for over a year, so it will be difficult to get them up and running again. I've created a [follow-up](https://github.com/elastic/kibana/issues/147489) to address this. Nice find!